ArduCopter/UserCode.cpp

```diff
@@ -42,3 +42,21 @@ void Copter::userhook_SuperSlowLoop()
     // put your 1Hz code here
 }
 #endif
+
+#ifdef USERHOOK_AUXSWITCH
+void Copter::userhook_auxSwitch1(uint8_t ch_flag)
+{
+    // put your aux switch #1 handler here (CHx_OPT = 47)
+}
+
+void Copter::userhook_auxSwitch2(uint8_t ch_flag)
+{
+    // put your aux switch #2 handler here (CHx_OPT = 48)
+}
+
+void Copter::userhook_auxSwitch3(uint8_t ch_flag)
+{
+    // put your aux switch #3 handler here (CHx_OPT = 49)
+}
+
+#endif
```

Comment:
miss new line at the end of file

Reply:
@khancyr ups, I never care about new lines at the EOF before. I believe I fixed it now.